internal/kgateway/extensions2/plugins/backend/ai/ai_model_cluster.go

```diff
@@ -164,13 +164,28 @@ func buildModelCluster(aiUs *v1alpha1.AIBackend, aiSecret *ir.Secret, multiSecre
 			},
 			Match: &structpb.Struct{},
 		},
-	}...)
-	out.LoadAssignment = &envoyendpointv3.ClusterLoadAssignment{
-		ClusterName: out.GetName(),
-		Endpoints:   prioritized,
 	}
 
-	return nil
+	// Build load assignment. Cluster name will be set at translation time.
```

Comment:
nit: move this comment inside ClusterLoadAssignment
```suggestion
	// ClusterName will be set at translation time.
```

Reply:
Done